src/main/java/Bob.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Scanner;
+
+public class Bob {
+
+    public static final String SEPARATOR = "____________________________________________________________";
+
+    public static void main(String[] args) {
+        String logo = "  ____        _     \n"
+                + " | |_) \\ ___ | |___  \n"
+                + " |  _ //  _  \\   _ \\ \n"
+                + " | |_)\\\\ (_) /  |_) |\n"
+                + " |____/ \\___/|_|___/ \n";
+        System.out.println(logo);
+        System.out.println("Hello! I'm Bob");
+        System.out.println("What can I do for you?");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
```

Comment:
Avoid magic numbers

Reply:
Great idea, thank you for your suggestion!